experimental/src/Config/SpanProcessor/NoopSpanProcessorConfig.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\Experimental\Config\SpanProcessor;
+
+use OpenTelemetry\Experimental\Config\ConfigInterface;
+use OpenTelemetry\Experimental\Config\SpanProcessorConfigInterface;
+
+class NoopSpanProcessorConfig implements ConfigInterface, SpanProcessorConfigInterface
+{
+    public function __construct(array $userConfig, array $environmentConfig)
+    {
+    }
```

Comment:
???

Reply:
It's because the constructor is defined in an interface...I will change it.

Reply:
@brettmc 
I know, this was just a lazy comment from me.  :)